Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -168,7 +167,7 @@ - (void)prepareLookup:(BTAPIClient *)apiClient completion:(void (^)(NSError * _N
         }
 
         if (configuration.cardinalAuthenticationJWT) {
-            self.threeDSecureV2Provider = [BTThreeDSecureV2Provider initializeProviderWithConfiguration:configuration
```

Comment:
I wasn't sure why a static function was initially used to return an instance in this deleted line.

Reply:
Not sure either! Makes more sense to have it as an `init` vs a method that is essentially an init. I am guessing there was a reason at some point but now we have the opportunity to streamline it!